tests/testthat.R

```diff
@@ -1,4 +1,5 @@
 library(testthat)
+library(patrick)
```

Comment:
Why `library()` if used only by namespace?

Reply:
make it clearer which package is needed by the test suite. I guess we could add a skip_if_not_installed to that file but the current approach is preferable to me

Reply:
OTOH `skip_if_not_installed()` would also be fine since the R CMD check workflows install all suggested packages so don't skip the tests.

Reply:
yes, fail fast is the goal. testthat and Patrick are on basically equal footing as deps as I see it.

the package will also be available on CRAN. using the namespace is mainly a polite way to convey to readers that this is not part of testthat, otherwise we could just use library and not namespace-qualify.

so I would either leave it as is, or use a skip (with patrick tests always at the bottom of a given file)

Reply:
Then let's go for a `skip`.